website/docs/r/vm_internal_disk.html.markdown

```diff
@@ -0,0 +1,107 @@
+  size_in_mb      = "13333"
+  bus_number      = 0
+  unit_number     = 1
+  storage_profile = "Development"
+  allow_vm_reboot = true
+  depends_on      = ["vcd_vapp_vm.web1"]
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `org` - (Optional) The name of organization to use, optional if defined at provider level. Useful when connected as sysadmin working across different organisations
+* `vdc` - (Optional) The name of VDC to use, optional if defined at provider level
+* `vapp_name` - (Required) The vApp this VM internal disk belongs to.
+* `vm_name` - (Required) VM in vApp in which internal disk is created.
+* `allow_vm_reboot` - (Optional) Powers off VM when changing any attribute of an IDE disk or unit/bus number of other disk types, after the change is complete VM is powered back on. Without this setting enabled, such changes on a powered-on VM would fail. Defaults to false.
+* `bus_type` - (Required) The type of disk controller. Possible values: `ide`, `parallel`( LSI Logic Parallel SCSI), `sas`(LSI Logic SAS (SCSI)), `paravirtual`(Paravirtual (SCSI)), `sata`. 
+* `size_in_mb` - (Required) The size of the disk in MB. 
+* `bus_number` - (Required) The number of the SCSI or IDE controller itself.
+* `unit_number` - (Required) The device number on the SCSI or IDE controller of the disk.
+* `iops` - (Optional) Specifies the IOPS for the disk. Default - 0.
+* `storage_profile` - (Optional) Storage profile which overrides the VM default one.
+
+## Attribute reference
+
+* `thin_provisioned` - Specifies whether the disk storage is pre-allocated or allocated on demand.
+
+## Importing
+
+~> **Note:** The current implementation of Terraform import can only import resources into the state. It does not generate
+configuration. [More information.][docs-import]
+
+An existing VM internal disk can be [imported][docs-import] into this resource via supplying its path.
+The path for this resource is made of org-name.vdc-name.vapp-name.vm-name.disk-id
+For example, using this structure, representing a VM internal disk that was **not** created using Terraform:
+
+```hcl
+resource "vcd_vm_internal_disk" "tf-myInternalDisk" {
+  org       = "my-org"
+  vdc       = "my-vdc"
+  vapp_name = "my-vapp"
+  vm_name   = "my-vm" 
+}
+```
+
+You can import such VM internal disk into terraform state using this command
+
+```
+terraform import vcd_vm_internal_disk.tf-myInternalDisk my-org.my-vdc.my-vapp.my-vm.my-disk-id
+```
+
+[docs-import]:https://www.terraform.io/docs/import/
+
+After importing, if you run `terraform plan` you will see the rest of the values and modify the script accordingly for
+further operations.
+
+### Listing VM internal disk IDs
+
+If you want to list IDs there is a special command **`terraform import vcd_vm_internal_disk.imported list@org-name.vcd-name.vapp-name.vm-name`**
+where `org-name` is the organization used, `vdc-name` is vDC name, `vapp-name` is vAPP name and `vm_name` is VM name in that vAPP.
```

Comment:
Please review the PR for "vAPP" capitalization and adjust to "vApp".

Reply:
done